app/Core/Setup.php

```diff
@@ -15,4 +15,24 @@ public function setUploadTypes(array $mimes): array
 
         return $mimes;
     }
+
+    /**
+     * @action login_enqueue_scripts
+     */
+    public function setLoginLogo()
+    {
+
+        if (! empty($logo = get_field('site', 'options')['logo'])) {
```

Comment:
Please read the current readme. We should use `firestarter()->settings()->get('')` for accessing the options.